src/pages/index.tsx

```diff
@@ -16,23 +15,13 @@ const IndexPage: NextPage = () => {
     };
 
     return (
-        <>
-            <MainLayout>
-                <div className={styles.container}>
-                    <h1 className="text-3xl font-bold underline">
-                        Simple testing test
-                    </h1>
-                    <button data-testid="api-button" onClick={callTestAPI}>
-                        Make API call
-                    </button>
-                    {renderedText != '' ? (
-                        <p data-testid="api-output">{renderedText}</p>
-                    ) : (
-                        <p />
-                    )}
-                </div>
-            </MainLayout>
-        </>
+                <MainLayout>
```

Comment:
These changes are breaking the existing tests and causing the build to fail. Since this page won't have any unique functionality, you can probably remove those tests.